src/main/java/uk/gov/hmcts/reform/jobscheduler/model/JobSchedulerPageRequest.java

```diff
@@ -5,14 +5,14 @@
 import org.springframework.data.domain.Sort;
 
 @JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
-public final class PageRequest extends org.springframework.data.domain.PageRequest {
+public final class JobSchedulerPageRequest extends org.springframework.data.domain.PageRequest {
 
     @Deprecated
```

Comment:
did you need to remove the `@Deprecated` annotation?

Reply:
Good point. I'm not sure as I was just renaming it for Sonar as it didn't like it having the same class name as the one it was extending in another package. Do u know why it was deprecated in the first place?

Reply:
It's a way of saying "please do not use this method because we are deprecating this in near future". and it is taken/overridden from super class. trying to hack in snake case strategy in spring paginated data response